dev/io.openliberty.org.eclipse.microprofile/metrics.3.0.bnd

```diff
@@ -21,6 +21,6 @@ Export-Package: \
   org.eclipse.microprofile.metrics.annotation;version=3.0
 
 Include-Resource: \
-  @${repo;org.eclipse.microprofile.metrics:microprofile-metrics-api;3.0.0.RC3;EXACT}
+  @${repo;org.eclipse.microprofile.metrics:microprofile-metrics-api;3.0}
```

Comment:
Do you know whether this will download from your sonatype staging?  Looks like it might be controlled in https://github.com/OpenLiberty/open-liberty/blob/integration/dev/settings.gradle

Reply:
This isn't downloading directly form sonatype staging.
The staging artifacts were downloaded and uploaded to IBM artifactory and IBM DHE.
Both scenarios were tested.